src/application/environment.rs

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
This module can be rename `env` for shortness.
It’s probably easier to deal with `std::path::Path` in this module.

Reply:
Renamed to ```env``` ```std::path::Path``` was much easier than what I was doing (still pretty new to rust) thank you for pointing that out!

Reply:
The whole module can deal with `std::path::Path`, you can modify the `tasks::List::load_file` to accept `std::path::Path`, to be exact a `AsRef<Path>` as [std::fs::File::open](https://doc.rust-lang.org/std/fs/struct.File.html#method.open).